app/timetables/models.py

```diff
@@ -22,7 +22,7 @@ class Meal(SlugifyMixin, models.Model):
     is scheduled to be served. E.g breakfast, lunch, etc.
     """
 
-    name = models.CharField(max_length=60)
+    name = models.CharField(max_length=60, verbose_name='Meal name')
```

Comment:
Thanks. It would be good to have all the words of the verbose_name start with letters in caps. For this, we would have `Meal Name` instead. Same for the rest.